frontend/src/container/QueryBuilder/filters/QueryBuilderSearch/OptionRenderer.tsx

```diff
@@ -3,25 +3,23 @@ import './QueryBuilderSearch.styles.scss';
 import { Tooltip } from 'antd';
 
 import { TagContainer, TagLabel, TagValue } from './style';
-import { getOptionType } from './utils';
 
 function OptionRenderer({
 	label,
 	value,
 	dataType,
+	type,
 }: OptionRendererProps): JSX.Element {
-	const optionType = getOptionType(label);
```

Comment:
get the type from the option itself rather than adding to prefix and then removing from prefix and deriving the same. opens up the gates to issues!